### PR TITLE
fix(examples): wrong properties in layers json files

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
Layers weren't displayed.
On wms `REGIONS.2017`, there's network error that is currently being resolved

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,11 +39,12 @@
             itowns.Fetcher.json('examples/layers/JSONLayers/Ortho.json').then(createWMTSSourceFromConfig).then(addColorLayerFromConfig);
             itowns.Fetcher.json('examples/layers/JSONLayers/OrthosCRS.json').then(createWMTSSourceFromConfig).then(addColorLayerFromConfig);
             itowns.Fetcher.json('examples/layers/JSONLayers/ScanEX.json').then(createWMTSSourceFromConfig).then(addColorLayerFromConfig);
-            itowns.Fetcher.json('examples/layers/JSONLayers/Region.json')
-                .then(function _(config) {
-                    config.source = new itowns.WMSSource(config.source);
-                    return config;
-                }).then(addColorLayerFromConfig);
+            // Error network is currently being resolved
```
Comment (zarov) [inline index.html]: If you updated the layer, why commenting it here ?

Comment (gchoqueux) [reply index.html]: I will solve the error.

Comment (gchoqueux) [reply index.html]: Fixed

**Review by zarov — APPROVED** [review]
Great, thanks !

**Pull request merged**